src/vos/vos_obj_cache.c

```diff
@@ -296,6 +271,37 @@ cache_object(struct daos_lru_cache *occ, struct vos_object **objp)
 	return 0;
 }
 
+static bool
+vos_obj_op_conflict(struct vos_object *obj, uint64_t flags, uint32_t intent, bool create)
```

Comment:
seems need not separate create flag, can just be "create = flags & VOS_OBJ_CREATE"?